pkgs/by-name/orca/package.nix

```diff
@@ -7,6 +7,10 @@
 , openssh
 , xtb
 , enableAvx2 ? true
+
+# Provide path to a custom script for otool_external
+, withCustomOtoolExternal ? false
+, customOtoolExternal ? null
```

Comment:
Should there maybe be an assert that `customOtoolExternal` is not null, if `withCustomOtoolExternal` is set?

Reply:
ok, now it should fail gracefully.